go.mod

```diff
@@ -68,6 +68,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/collectdreceiver v0.105.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver v0.105.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver v0.105.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/haproxyreceiver v0.106.1
```

Comment:
Should we maybe import `v0.105.0` to be consistent with versions here? This ends up upgrading other deps as well, my only concern being potential challenges with the upgrade deps PR that will need to be done for `v0.106.1`, but not sure if it would actually impact much.

Reply:
done